internal/ingress/controller/nginx_test.go

```diff
@@ -361,28 +361,28 @@ func TestCleanTempNginxCfg(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	tmpfile, err := os.CreateTemp("", tempNginxPattern)
+	tmpfile1, err := os.CreateTemp(tmpDir, tempNginxPattern)
```

Comment:
here you dont need to set that

```
If dir is the empty string, CreateTemp uses the default directory for temporary files, as returned by TempDir. 
```

Reply:
if we leave it unset, it uses `/tmp` but the point is that some time ago a change was introduced to `NewNGINXController` func that defined `tmpDir` for temp files (configs) as os.TempDir() + "/nginx", from that point `cleanTempNginxCfg` func had stopped to work.
Now, we are here :)

Reply:
i see, thanks for the clarification